### ISSUE TITLE
Multiline subtitles exported from MuseScore only show first line

### ISSUE THREAD

**Issue by bbayu123** [opensheetmusicdisplay/opensheetmusicdisplay#1503]
I have exported a MusicXML file from MuseScore 4.2.0, and noticed that only the first line of the subtitles (and composer) is shown. Is this something that can be fixed, or is the way MuseScore represents multiline credits different from the specification?

[test.musicxml.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/14030099/test.musicxml.zip)

MuseScore 4.2.0:
![image](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/26726381/ccd5f6c6-f73a-41fa-8fb4-f8faff275a7b)
OSMD 1.8.5:
![image](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/26726381/53ae70a0-329b-4ffe-9af0-01dc99d205a4)

**Comment by sschmidTU** [comment]
PR #1508 fixes this.
<img width="888" alt="image" src="https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/33069673/13ef357f-f87a-4c88-be85-4e1231e4da62">

But it might change which composers and subtitles appear in other scores as well, since they're now read primarily from `<credit-words>` instead of the `<identification>` and `<work>` tags.
So this will probably be released in the update after the next one (though both should be released on the same day).

The multiple `<credit-words>` inside one `<credit>` tag syntax seems to be correct MusicXML, which I didn't know and was not parsed correctly.

**Comment by sschmidTU** [comment]
Fixed in #1508.
This will be released in OSMD 1.8.6 shortly.

Info for others:
This changes what subtitles and composers are displayed for some XML files.
I recommend adapting your XML files if the changes are not desired.
But if you temporarily want to use the previous subtitle and composer parsing and display,
you can use this (deprecated) option (which will be removed soon):
```js
osmd.EngravingRules.SheetComposerSubtitleUseLegacyParsing = true;
```
